### PR TITLE
Docs: Fixed and moved JSfiddle demo to samples.

### LINKED ISSUE #17821: General docs, Colors: not working demo
The first demo on this website isn't working with the newest version of Highcharts: https://www.highcharts.com/docs/chart-design-and-style/colors

#### Live demo with steps to reproduce
https://jsfiddle.net/highcharts/zy1epj3o/

#### Product version
10.2.1

### PR DESCRIPTION
**Pull request by bre1470**
Fixed #17821

### CODE REVIEW

File: samples/highcharts/chart/colors-parsers/demo.details [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+ name: Highcharts Demo
+ authors:
+   - Torstein Hønsi
+ js_wrap: b
+ skipTest: true
```
Comment (TorsteinHonsi) [inline samples/highcharts/chart/colors-parsers/demo.details:6]: Remove `skipTest: true`, this sample should be part of the visual tests.

File: samples/highcharts/chart/colors-parsers/demo.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+Highcharts.Color.parsers.push({
+    regex: /^[a-z]+$/,
+    parse: function (result) {
+        var rgb = new RGBColor(result[0]);
```
Comment (TorsteinHonsi) [inline samples/highcharts/chart/colors-parsers/demo.js:4]: Use modern syntax, `const` and possibly arrow function.

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Thanks! Some best practice adjustments for the demo.

File: samples/highcharts/chart/colors-parsers/demo.details (hunk 1 above)
Comment (bre1470) [reply samples/highcharts/chart/colors-parsers/demo.details:6]: We then need to have RGBColor part of vendors.

Comment (bre1470) [reply samples/highcharts/chart/colors-parsers/demo.details:6]: You have to help me with that, I did not managed to make it work.

Comment (bre1470) [reply samples/highcharts/chart/colors-parsers/demo.details]: It looks like I figured it out this time.

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**